tests/infrastructure/test_kubernetes_job.py

```diff
@@ -922,7 +926,7 @@ def mock_stream(*args, **kwargs):
             yield {"object": job, "type": "ADDED"}
 
     def mock_log_stream(*args, **kwargs):
-        anyio.sleep(500)
```

Comment:
This wasn't actually calling `anyio.sleep` it was calling the mock above. I've attempted to make that less confusing by calling the mock directly.